101-media-services-create/azuredeploy.json

```diff
@@ -0,0 +1,44 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "mediaServiceName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of the Media Services account. A Media Services account name is globally unique, all lowercase numbers or letters with no spaces."
+      }
+    }
+  },
+  "variables": {
+    "storageName": "[concat('storage', uniqueString(resourceGroup().id))]",
+    "storageType": "Standard_LRS"
```

Comment:
I'd recommend eliminating the storageType variable and just setting the accountType inline: "accountType": "Starndard_LRS". That way if the user wants to change the value, they'll get suggestions on valid options via schema validation (in editors like VS and VSCode)